### DISCUSSION TITLE
Fatal error: Call to undefined function opcache_reset() after changing theme

### DISCUSSION THREAD

**Discussion by oliworx** [datenstrom/community#258] [Report a bug] [answered]
After uploading the paris theme I get this error:

Fatal error: Uncaught Error: Call to undefined function opcache_reset() in /homepages/28/d13362316/htdocs/yellow-b/system/plugins/update.php:297 Stack trace: #0 /homepages/28/d13362316/htdocs/yellow-b/system/plugins/update.php(557): YellowUpdate->updateSoftware() #1 /homepages/28/d13362316/htdocs/yellow-b/system/plugins/update.php(143): YellowUpdate->processRequestInstallationPending('http', 'demo-b.usc-muen...', '', '/', 'content/1-home/...') #2 /homepages/28/d13362316/htdocs/yellow-b/system/plugins/core.php(136): YellowUpdate->onRequest('http', 'demo-b.usc-muen...', '', '/', 'content/1-home/...') #3 /homepages/28/d13362316/htdocs/yellow-b/yellow.php(13): YellowCore->request() #4 {main} thrown in /homepages/28/d13362316/htdocs/yellow-b/system/plugins/update.php on line 297

**Comment by oliworx** [comment, marked as answer]
The function is probably blacklisted on my webspace.
I changed the line to
`if (function_exists('opcache_reset') ) { opcache_reset();}`
and it seems to be ok now.

**Comment by markseuffert** [comment]
Thanks for the bug report.

Sorry about that, I added the check to the latest version. Should be working now.
